modules/hub/src/ChainsawService.test.ts

```diff
@@ -154,13 +153,22 @@ describe('ChainsawService::mocked Web3', function() {
   })
 
   describe('processSingleTx', () => {
-    it('should return poll type "SKIP_EVENTS" if state generation fails', async () => {
+    it('should return poll type "RETRY" if state generation fails', async () => {
      const pollType = await cs.processSingleTx(failedTxHash)
-     assert.equal(pollType, 'SKIP_EVENTS')
+     assert.equal(pollType, 'RETRY')
```

Comment:
all of the other poll types have the suffix `_EVENTS`, so i think we should add it here for consistency